distr_tracing/distr_tracing_install/distr-tracing-deploying-otel.adoc

```diff
@@ -9,3 +9,4 @@ toc::[]
 The {OTELName} Operator uses a custom resource definition (CRD) file that defines the architecture and configuration settings to be used when creating and deploying the {OTELName} resources. You can either install the default configuration or modify the file to better suit your business requirements. 
 
 include::modules/distr-tracing-config-otel-collector.adoc[leveloffset=+1]
+include::modules/distr-tracing-config-tempo-spanmetrics.adoc[leveloffset=+1]
```

Comment:
The content belongs to the Tempo documentation not OTEL. But I am putting it here temporarily because there are no Tempo docs at this moment